daemon/algod/api/server/v2/dryrun.go

```diff
@@ -119,45 +119,27 @@ type dryrunDebugReceiver struct {
 }
 
 func (ddr *dryrunDebugReceiver) updateScratch() {
-	any := false
 	maxActive := -1
 	lasti := len(ddr.history) - 1
 
 	if ddr.history[lasti].Scratch == nil {
 		return
 	}
 
+	ddr.scratchActive = make([]bool, 256)
```

Comment:
maybe we do not need to realloc at each call and reuse existing array? Just add `ddr.scratchActive[i] = false` outside of the `if` condition in the loop below